parlai/mturk/core/mturk_data_handler.py

```diff
@@ -443,6 +470,16 @@ def get_worker_assignment_pairing(self, worker_id, assignment_id):
             results = c.fetchone()
             return results
 
+    def get_all_run_data(self, start=0, count=30):
+        '''get all the run data for all task_group_ids.'''
```

Comment:
nit: for all of our docstrings elsewhere we use double quotation marks

Reply:
Oops, good catch. Fixing!